features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1969,4 +1969,9 @@
         <ApplicationName>My Account</ApplicationName>
     </SystemApplications>
 
+    <!-- Claim to store localization code  -->
+    <!--    By setting true localization code will be stored inside the http://wso2.org/claims/locality-->
+    <!--    By setting false localization code will be stored inside the http://wso2.org/claims/local-->
+    <isLegacyLocalityClaimUri>true</isLegacyLocalityClaimUri>
```

Comment:
shall we use to `<useLegacyLocalizationClaim>` ?

Reply:
also the default value should be false.

Reply:
Fixed